frontend/server/src/DAO/Users.php

```diff
@@ -281,8 +281,60 @@ public static function getStatusVerified(string $email) {
         /** @var null|array{username: string, verified: bool} */
         return \OmegaUp\MySQLConnection::getInstance()->GetRow($sql);
     }
+      /**
+       * Delete User
+       */
 
```

Comment:
```suggestion

    /**
     * Delete User
     */
```

docstrings should have no empty lines between them and what they are describing. also the `/` should be aligned with the start of the declaration.

Reply:
okay got it!

Reply:
this is still missing.